packages/react-admin/src/actions/authActions.js

```diff
@@ -23,7 +23,17 @@ export const userCheck = (payload, pathName, routeParams) => ({
 
 export const USER_LOGOUT = 'RA/USER_LOGOUT';
 
-export const userLogout = () => ({
+/**
+ * Action to trigger logout of the current user. The entire redux state will be cleared
+ * thanks to the resettableAppReducer in Admin.
+ * @see: Admin.js
+ * @param redirectTo Path to direct to after logout
+ * @return {{type: string, payload: {redirectTo: string}, meta: {auth: boolean}}}
```

Comment:
what's the double curly brace for?

Reply:
https://github.com/Microsoft/TypeScript/wiki/JSDoc-support-in-JavaScript
Jsdoc allows to describe the return object inline.

Reply:
Gotcha